tests/src/core/importType.js

```diff
@@ -31,6 +31,7 @@ describe('importType(name)', function () {
   it("should return 'external' for scopes packages", function() {
     expect(importType('@cycle/core', context)).to.equal('external')
     expect(importType('@cycle/dom', context)).to.equal('external')
+    expect(importType('@some-thing/something', context)).to.equal('external')
```

Comment:
Could you add additional tests for the following cases? (and more if you think of some, better safe than sorry)
- `@something/some-thing`
- `@something/something/something`